tests/create_sg/main.tf

```diff
@@ -0,0 +1,27 @@
+provider aws {
+  region = "us-east-1"
+}
+
+module "vpc" {
+  source = "github.com/terraform-aws-modules/terraform-aws-vpc?ref=v2.15.0"
```

Comment:
I know dependabot isn't working with 0.12 yet, but let's add the test directory to the dependabot config anyway, so when it does start working, we won't have to go back and make more changes to get it to update these version refs...

Reply:
Done